socorro/middleware/search_common.py

```diff
@@ -278,7 +278,8 @@ def fix_date_parameter(self, parameters):
                         msg='date must have a prefix operator'
                     )
                 today = datetimeutil.utc_now()
-                if param.value > today:
+                # We add a safety margin to not be too strict.
+                if param.value > today + datetime.timedelta(days=1):
```

Comment:
Why a whole day? Why not one minute. 
The granularity of the datepickers on SuperSearch is minute. I just tested it and it correctly set the time to the current hour and minute in UTC. 
Not possible to reproduce. 
If your computer's clock is a couple of seconds *ahead* it could happen that the minute number is 1 minute greater than the UTC standard time. So, it would be safe to just add the margin of 1 minute. 

If someone has a computer whose time is **more** than 60 seconds too fast, that would be super rare. 

The reason we have these validation guards is to that there aren't ES errors when trying to query an index that doesn't exist *yet*. If someone has a crazy date on their computer, and we allow it in the validation, they could easily have a hard 500 Internal Server Error because we queried an index based on a future date. 

So, in conclusion:
* Why not just 1 minute?
* Why not generate that default date from the server instead of letting `new Date()` in JS do it?